src/metabase/sync/sync_metadata/sync_timezone.clj

```diff
@@ -3,15 +3,14 @@
    [java-time :as t]
    [metabase.driver :as driver]
    [metabase.driver.util :as driver.u]
-   [metabase.lib.schema.expression.temporal
-    :as lib.schema.expression.temporal]
    [metabase.sync.interface :as i]
    [metabase.util.malli :as mu]
    [toucan2.core :as t2]))
 
 (set! *warn-on-reflection* true)
 
-(mu/defn sync-timezone! :- [:maybe [:map [:timezone-id [:maybe ::lib.schema.expression.temporal/timezone-id]]]]
```

Comment:
instead of changing the schema here, how about we loosen the `lib.schema.expression.temporal/timezone-id` schema to accept the short name as well?

Reply:
I don't think we want to expand `lib.schema.expression.temporal/timezone-id` because it is used in other places such as [MBQL schema](https://github.com/metabase/metabase/blob/c47e63c34d9f37f01af91a76b0246a9075ec98ff/shared/src/metabase/mbql/schema.cljc#L99) where we want to limit values to the set returned by `(java.time.ZoneId/getAvailableZoneIds)`.

Reply:
Actually, probably the best solution is to expand just the return type of `sync-timezone!` like this:
[Expand sync-timezone!'s return type to include offset strings](https://github.com/metabase/metabase/pull/34055/commits/267b387a885bdb392b00365933974fcfc371ab1d)

Reply:
In https://github.com/metabase/metabase/pull/32686/files#diff-7adb5f5a6daa500e11dae570c20d6a4865f2be4db49fa772179cb1f906c4e718 I do in fact extend `timezone-id`. That PR is waiting for @camsaul to have a final look.

Reply:
Interesting. `timezone-id` is used as the schema for arguments to the `:convert-timezone` clause, so if you expand the type you should also test `:convert-timezone` works correctly with offsets for all drivers. I suspect you'll find it won't.

Reply:
[Here's](https://github.com/metabase/metabase/blob/481604aef6a7c007782b62123115e35d516a8af9/test/metabase/query_processor_test/date_time_zone_functions_test.clj#L525) the `convert-timezone` tests. We currently only test arguments with time zone IDs, not offsets